### DISCUSSION TITLE
Case Study Bug.  stochastic_optimal_powerflow.jl

### DISCUSSION THREAD

**Discussion by ChengweiLou** [infiniteopt/InfiniteOpt.jl#242] [Q&A] [answered]
julia> include("stochastic_optimal_powerflow.jl")
Set parameter Username
Academic license - for non-commercial use only - expires 2022-04-29
Set parameter Username
Academic license - for non-commercial use only - expires 2022-04-29
Model 1 infeasible with α = 0.975
┌ Warning: `vendor()` is deprecated, use `BLAS.get_config()` and inspect the output instead
│   caller = npyinitialize() at numpy.jl:67
└ @ PyCall ~/.julia/packages/PyCall/L0fLP/src/numpy.jl:67
PyObject Text(0, 0.5, 'Probability Level ($\\alpha$)')

I run the cases from https://github.com/zavalab/JuliaBox/tree/master/InfiniteDimensionalCases/CaseStudy1
However, I cannot get the correct result.

Thank you.

**Comment by pulsipher** [comment, marked as answer]
Julia has changed how its random numbers are generated, so the models in this example no longer used the same supports (i.e., random samples) and thus got different answers. If you try the updated version now it should work again.
